config/delta-consumers/op-consumer/mapping/queries/op2dl/mapping/bestuurseenheid.sparql

```diff
@@ -0,0 +1,10 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+CONSTRUCT {
+  ?s skos:topConceptOf <http://lblod.data.gift/concept-schemes/7e2b965e-c824-474f-b5d5-b1c115740083> ;
+    skos:inScheme <http://lblod.data.gift/concept-schemes/7e2b965e-c824-474f-b5d5-b1c115740083> .
+} WHERE {
+  ?s a besluit:Bestuurseenheid;
+     <http://www.w3.org/ns/org#classification> <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/5ab0e9b8a3b2ca7c5e000001>.
```

Comment:
The concept scheme should also be added to provincies no? I checked quickly with the following query, looks like it's currently added to worship units, gemeente and provincie (and ABB but I suppose it's a special case haha)
```
select distinct ?classif where { ?s skos:inScheme <http://lblod.data.gift/concept-schemes/7e2b965e-c824-474f-b5d5-b1c115740083> ; <http://www.w3.org/ns/org#classification>/skos:prefLabel ?classif . }
```